### PR TITLE
added http status code 419

### PR DESCRIPTION
**Pull request by trirpi**

### CODE REVIEW

**Review by casdr — APPROVED** [review]

File: bot.php [hunk 1]
```diff
@@ -48,7 +48,7 @@
 
         // Random chat triggers
         $message = trim($message);
-        if(($message=="ok"||$message=="oke") && rand(1,5)==4) output("is goed");
+        if(($message=="ok"||$message=="oke") && rand(1,4)==2) output("is goed");
```
Comment (casdr) [inline bot.php:51]: This shouldn't be in this PR

**Review by casdr — CHANGES_REQUESTED** [review]

File: bot.php (hunk 1 above)
Comment (trirpi) [reply bot.php:51]: I thought that was another PR. But now it is in the same one.

Comment (trirpi) [reply bot.php:51]: ja dan zal k dat doen zkers

Comment (trirpi) [reply bot.php:51]: ff kijken hoe

**Comment by casdr** [comment]
Closed in favor of #21

**Pull request closed without merging**